src/Cairo.jl

```diff
@@ -1,4 +1,4 @@
-VERSION < v"0.7.0-beta2.199" && __precompile__()
```

Comment:
why are you disabling precompilation on 0.6?

Reply:
It doesn't look like my idea ...

Reply:
I deleted the release and i will look at this again tomorrow morning (my time).

Reply:
Believe me or not but I have no idea why I removed that. Maybe I hit `dd` too hastily in vim. But it was certainly not my intention.